### PR TITLE
gather_method = "max" or "mean" of frequencies in a band for spectral…

### PR DESCRIPTION
**Pull request by davidmeunier79**
… connectivity

### CODE REVIEW

File: ephypype/tests/test_spectral.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+ts_mat = np.random.rand(nb_ROI, time_length) * \
+    np.random.choice([-1, 1], size=(nb_ROI, time_length))
+
+ts_mat_trials = np.random.rand(nb_trials, nb_ROI, time_length) * \
+    np.random.choice([-1, 1], size=(nb_trials, nb_ROI, time_length))
+
+
+def test_compute_spectral_connectivity_one_trial():
+
+    """Test compute_spectral_connectivity plv with one trial."""
```
[the remark this answers is not in the document]
Comment (davidmeunier79) [reply ephypype/tests/test_spectral.py]: modified in the new version

**Comment by davidmeunier79** [comment]
not sure what is wrong here, I did not touch test_preproc ....

File: ephypype/spectral.py [hunk 2]
```diff
@@ -2,36 +2,56 @@
...
 Author: David Meunier <david_meunier_79@hotmail.fr>
 """
+import os
+
+import numpy as np
+from scipy.io import savemat
+
+from mne.connectivity import spectral_connectivity
+from mne.viz import circular_layout, plot_connectivity_circle
+import matplotlib.pyplot as plt
```
Comment (jasmainak) [inline ephypype/spectral.py]: The `matplotlib` import should not be at the top of the file

File: ephypype/tests/test_spectral.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+def test_compute_spectral_connectivity_mean_max():
+    """Test compute_spectral_connectivity mean and max."""
+    res_mean = compute_spectral_connectivity(data=ts_mat_trials,
+                                             con_method="coh",
+                                             mode="multitaper", fmin=fmin,
+                                             fmax=fmax, sfreq=sfreq,
+                                             gathering_method='mean')
+
+    print(res_mean)
```
Comment (jasmainak) [inline ephypype/tests/test_spectral.py]: can you remove these `print` statements? They should not be in tests.

**Comment by davidmeunier79** [comment]
@jasmainak @annapasca  @dmalt @EtienneCmb can someone please accept this PR?

**Comment by jasmainak** [comment]
Merged. Thanks @davidmeunier79 !

**Pull request merged**